jobs/build/appregistry/appregistry.groovy

```diff
@@ -348,6 +352,7 @@ def stageAttachMetadata(operatorBuilds) {
         ${params.DRY_RUN ? "--noop" : ""}
     """
 
+    cleanAdvisoryBuilds(params.METADATA_ADVISORY)
```

Comment:
As @joepvd mentioned, if we only rebuild specific metadatas, this is going to remove all metadata builds. Would like to have a smarter way (e.g. filter by environment name then remove them with elliott)

Reply:
Or a simple approach: add an option to opt out this behavior if checked.